Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -880,11 +883,12 @@ def SwapKeepAndDrop(l):
 from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
 from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
 from Configuration.Eras.Modifier_phase2_muon_cff import phase2_muon
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
 for _entry in [FEVTDEBUGEventContent,FEVTDEBUGHLTEventContent,FEVTEventContent]:
     run2_GEM_2017.toModify(_entry, outputCommands = _entry.outputCommands + ['keep *_muonGEMDigis_*_*'])
     run3_GEM.toModify(_entry, outputCommands = _entry.outputCommands + ['keep *_muonGEMDigis_*_*'])
     phase2_muon.toModify(_entry, outputCommands = _entry.outputCommands + ['keep *_muonGEMDigis_*_*'])
-
+    pp_on_AA_2018.toModify(_entry, outputCommands = _entry.outputCommands + ['keep FEDRawDataCollection_rawDataRepacker_*_*'])
```

Comment:
why does this need to be handled separately from the rest?

Reply:
@davidlange6 do you mean not modifying directly the RAW content?

Reply:
there is no simple obvious alternative that's clearly not going to break anything outside of pp_on_AA

Reply:
@slava77 the initial simple attempt to modify RAW was not working...

Reply:
> @slava77 the initial simple attempt to modify RAW was not working...

I'm not sure if this is connected with David's comment

Reply:
@slava77 well, the modification tried was just for pp_on_AA, changing the base definition is different. I do not see how this change might break other things, and I have checked that it solves the observed problem, so in principle it may be merged if a fast patch is needed